.circleci/config.yml

```diff
@@ -548,11 +573,22 @@ workflows:
               only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
             branches:
               only: unsigned
+      - build-ios-iphonesimulator:
+          context: mattermost-mobile-unsigned
```

Comment:
I only assumed that this context can be used since it's mostly the same when building unsigned apps.